content/posts/newsletter/2020-04-02.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Draft Title - Date
+save_as: tba
+url: tba
+date: 2020-04-02 23:40
+author: emus
+blog_id: newsletter
+category: Newsletter
+
+---
+
+# Collaboration over the Internet! XX May 2020
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of April 2020.
+
+We are always happy for contributors - just come along and talk to us in the [Comm-Team MUC](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort, which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+[Subscribe](https://tinyletter.com/xmpp) to receive the next edition in your inbox!
+
+
+#### Newsletter translations
+
+The translations of the XMPP Newsletter will be release here:
+
+- [https://www.jabber.de/ German]
+- [https://news.jabberfr.org/category/newsletter/ French]
```

Comment:
news.jabber.fr is apparently out of order, and did not publish translations since 09/2019
I'm used to publish my french translations to https://linuxfr.org/tags/xmpp/public